src/table/sprites.h

```diff
@@ -290,8 +290,10 @@ static const SpriteID SPR_TRAMWAY_DEPOT_NO_TRACK       = SPR_TRAMWAY_BASE + 113;
 static const uint16 TRAMWAY_SPRITE_COUNT = 119;
 
 /** One way road sprites */
-static const SpriteID SPR_ONEWAY_BASE = SPR_TRAMWAY_BASE + TRAMWAY_SPRITE_COUNT;
-static const uint16 ONEWAY_SPRITE_COUNT = 6;
+static const SpriteID SPR_ONEWAY_BASE           = SPR_TRAMWAY_BASE + TRAMWAY_SPRITE_COUNT;
+static const SpriteID SPR_ONEWAY_SLOPE_N_OFFSET = 6;
+static const SpriteID SPR_ONEWAY_SLOPE_S_OFFSET = 12;
```

Comment:
Are these names for North and South? Should they be for axis instead?

Reply:
Yes and no respectively. The sprites are grouped by (flat) (north-facing slopes) (south-facing slopes), with each of those corresponding closely to the pre-existing set of sprites.

Grouping them by axis would make it more complicated to retain compatibility with existing basesets and grfs.

See oneway.png in the OpenGFX or NML PRs.